### ISSUE TITLE
this has been removed from the webstore

### ISSUE THREAD

**Issue by markifi** [rubenmv/chrome-extension-tabs-queue#19]
https://chrome.google.com/webstore/detail/tabs-limiter-with-queue/kaamkonbephafcojgajnbgbdgbgbboap

no longer available

**Comment by yuseinishiyama** [comment]
You still can re-enable it in `chrome://extensions` if you have it already installed though I am warned that `This extension violates the Chrome Web Store policy.`

**Comment by raqua** [comment]
Yes, you can, but it will be disabled again next day. It would be helpful if Google actually gave any reason for this. I would like to know if there extension is doing something nefarious or it is just some new Google policy that is not really relevant to me and I can safely ignore it.

**Comment by markifi** [comment]
with only 3 gigs of ram, I already miss this extension a lot the last two
days

**Comment by rubenmv** [comment]
Hi everyone, the extension was taken down by Google, I received this message:

![image](https://user-images.githubusercontent.com/64125/120040307-965dd280-c006-11eb-9971-7a76e81fc9e1.png)

I don't know what are they even talking about, the functionality they mention is the basic functionality of the extension. I haven't used it in a long time, but I just tested it and it seems to be working fine.

I just submitted a new version, if they reject it again I'll try and get in touch with support.

**Comment by bit-recycling** [comment]
Maybe they didnt get the part with "tabs per window" and tested against "tabs per chrome instance"?
Just a guess, since the extension description in my browser just says "Tabs limiter with queue."

**Comment by markifi** [comment]
any update on this from google?

**Comment by markifi** [comment]
i don't know why they destroyed this, but it's really frustrating to be without it

**Comment by yuseinishiyama** [comment]
So it seems that the new submission was unsuccessful. Please let me know if there's anything I can help with. I now realise how essential this extension is to be productive... (in fact, I still can install this from the source so I won't be too needy here)

**Comment by AyoubBBK** [comment]
> So it seems that the new submission was unsuccessful. Please let me know if there's anything I can help with. I now realise how essential this extension is to be productive... (in fact, I still can install this from the source so I won't be too needy here)

How can I do the same ? I mean installing the extension from the source code ! thanks in advance.

**Comment by markifi** [comment]
does anyone know of an adequate replacement to this? there are dozens of seemingly good tab limiter and lazy load extensions, none of which seem to be doing what this does, namely letting you open 200 tabs from your bookmarks and *immediately* suspending them into a queue you can work through the next 2 hours.

**Comment by yuseinishiyama** [comment]
@AyoubBBK Basically this https://dev.to/ben/how-to-install-chrome-extensions-manually-from-github-1612 Download the code from GitHub and load `src` directory (at your own risk!)

**Comment by lumeng** [comment]
2.0.5.zip in the release/ directory seems broken. It can not be extracted.

**Comment by markifi** [comment]
Tab Limiter And Restorer by InzkDev is a possible replacement, but it's nowhere near as good yet
https://chrome.google.com/webstore/detail/tab-limiter-and-restorer/ifhilckbhneppbcbpkimbgeahhijhdpj/related

edit: this extension has been improved and as far as i'm concerned it's now better than tab limiter with queue used to be

**Comment by meganstrickland** [comment]
Any news on getting this extension re-approved by the Chrome store? Is there anything users can do to help?

**Comment by pidgeon777** [comment]
I'm also interested.

**Comment by aspiers** [comment]
It's really depressing how shit Google is with stuff like this, and how much of a walled garden / dictatorship they've built in the web world.  This is why Open Source and decentralized technology is so important.

Anyway, regarding this suggested alternative:

> Tab Limiter And Restorer by InzkDev is a possible replacement, but it's nowhere near as good yet [chrome.google.com/webstore/detail/tab-limiter-and-restorer/ifhilckbhneppbcbpkimbgeahhijhdpj/related](https://chrome.google.com/webstore/detail/tab-limiter-and-restorer/ifhilckbhneppbcbpkimbgeahhijhdpj/related)
> 
> edit: this extension has been improved and as far as i'm concerned it's now better than tab limiter with queue used to be

I can't see any link to a website or source code for that extension, which unless I'm missing something means that it can't be easily verified that it adheres to the claimed privacy policy.  For a browser extension handling personal browsing history, this just isn't good enough for me and for many people.  One of the big advantages of rubenmv's extension is that the source is not only right here on GitHub, but it's also GPL which means that everyone can work together to maintain and improve it.

**Comment by aspiers** [comment]
In the absence of Google coming to their senses, could we at least have downloadable releases as `.zip` files here on GitHub?  This can even be automated, e.g. see https://github.com/fregante/browser-extension-template/blob/02fc1a03dd59bb77d4934971ef3f2d0aad360994/.github/workflows/release.yml#L42-L44

**Comment by aspiers** [comment]
Oh, just noticed that the `release_history/` subdirectory has `.zip` releases.  Would be good if that was documented in the README at least.

**Comment by pidgeon777** [comment]
In the meanwhile I'm trying to keep as few tabs as possible open. Less distractions, less procastination, more RAM.
